api/runner/runner.go

```diff
@@ -29,6 +29,7 @@ type Runner struct {
 	flog         FuncLogger
 	availableMem int64
 	usedMem      int64
+	Tasks        chan task.Request
```

Comment:
@ccirello I don't see why this change really prevent intercepting the communication between server and runner. Like you you've noticed the channel is still visible and can be easily replaced externally. 
If you take a look on tasks no change has been done to the test it self, we are currently testing the same way as before, but the channel creation is no longer exposed.
I see this as benefit because all packages the were using this channel also uses the `runner` and this channel is very related to the `runner`. So why not embedding that to it?
I think hiding this channel inside the runner you keep the intercept-ability but also reduce the complexity for the user when he wants to creates his own custom IronFunctions server.